### PR TITLE
Add root version option

### PR DESCRIPTION
**Pull request by borisdamevin**

### CODE REVIEW

File: lib/prepare.test.js [hunk 1]
```diff
@@ -146,6 +146,47 @@ it("Update lerna.json and root package.json when one or more package has changed
 	expect(context.log).toHaveBeenCalledWith("Write version %s to lerna.json in %s", "1.0.0", cwd);
 });
 
+it("Update only lerna.json when one or more package has changed", async () => {
```
Comment (ext) [inline lib/prepare.test.js:149]: Maybe this description should contain something about `rootVersion` being `false`?

File: lib/prepare.test.js [hunk 2]
```diff
@@ -146,6 +146,47 @@ it("Update lerna.json and root package.json when one or more package has changed
...
+	);
+
+	// Verify the logger has been called with the version updated
+	expect(context.log).toHaveBeenCalledWith("1 package need version bump: [ 'foo' ]");
+	expect(context.log).toHaveBeenCalledWith(
+		"Write version %s to package.json in %s",
+		"1.0.0",
+		pkg.location
+	);
+	expect(context.log).toHaveBeenCalledWith("Write version %s to lerna.json in %s", "1.0.0", cwd);
```
Comment (ext) [inline lib/prepare.test.js:187]: Add an expect to ensure `package.json` has not been updated?

**Review by ext — APPROVED** [review]
Looks good to me, thank you for the PR. If you're ready I can merge and release your changes right away.

**Comment by borisdamevin** [comment]
When you want! @ext

**Pull request merged**